### ISSUE TITLE
GCloud deploy fails on Windows 10 using the FAKE script

### ISSUE THREAD

**Issue by garethhubball** [SAFE-Stack/SAFE-template#245]
I've just created a new application with `dotnet new safe -d gcp-appengine -c remoting` and then tried to deploy it. The `gcloud` tool and Google Cloud SDK are correctly installed and setup on my computer, however FAKE fails with the following:

```Starting target 'Deploy'
.> "gcloud" app deploy --quiet (In: false, Out: false, Err: false)
Finished (Failed) 'Deploy' in 00:00:00.0047672

---------------------------------------------------------------------
Build Time Report
---------------------------------------------------------------------
Target          Duration
------          --------
Clean           00:00:00.0669205
InstallClient   00:00:01.7335822
Build           00:00:23.6078246
Bundle          00:00:01.6690944
Deploy          00:00:00.0044250   (Start of process 'gcloud' failed.)
Total:          00:00:27.1818635
Status:         Failure
---------------------------------------------------------------------
saving cache...
Script reported an error, see standard error for details.
Script reported an error:
-> BuildFailedException: Target 'Deploy' failed.
   StackTrace:
        at Fake.Core.TargetModule.raiseIfError(OptionalTargetContext context) in D:\a\1\s\src\app\Fake.Core.Target\Target.fs:line 820
        at Fake.Core.TargetModule.runOrDefaultWithArguments(String defaultTarget) in D:\a\1\s\src\app\Fake.Core.Target\Target.fs:line 940
        at <StartupCode$build_E3EF8E176AA9511D8D28025310B103B8240B996BC99D20058A828538954D0EA5>.$Build$fsx.main@() in C:\Users\ridec\Code\OldManTeacher\build.fsx:line 148
-> One or more errors occurred. (Start of process 'gcloud' failed.)
-> Start of process 'gcloud' failed.
   StackTrace:
        at Fake.Core.Process.processStarter@409.Invoke(RawCreateProcess c, Process p) in D:\a\1\s\src\app\Fake.Core.Process\Process.fs:line 421
        at Fake.Core.RawProc.start@188(FSharpFunc`2 startProcessRaw, RawCreateProcess c, ProcessStartInfo p, StreamSpecs streamSpec, Process toolProcess, CancellationTokenSource tok, FSharpRef`1 isStarted, FSharpRef`1 readOutputTask, FSharpRef`1 readErrorTask, FSharpRef`1 redirectStdInTask, Unit unitVar0) in D:\a\1\s\src\app\Fake.Core.Process\RawProc.fs:line 261        
        at Fake.Core.RawProc.syncStart@236.Invoke(Unit unitVar0)
        at Microsoft.FSharp.Control.AsyncResult`1.Commit()
        at Microsoft.FSharp.Control.AsyncPrimitives.RunSynchronouslyInCurrentThread[a](CancellationToken cancellationToken, FSharpAsync`1 computation)
        at Microsoft.FSharp.Control.AsyncPrimitives.RunSynchronously[T](CancellationToken cancellationToken, FSharpAsync`1 computation, FSharpOption`1 timeout)
        at Microsoft.FSharp.Control.FSharpAsync.RunSynchronously[T](FSharpAsync`1 computation, FSharpOption`1 timeout, FSharpOption`1 cancellationToken)
        at Build.runTool(String cmd, String args, String workingDir) in C:\Users\ridec\Code\OldManTeacher\build.fsx:line 36
        at Build.clo@129-8.Invoke(TargetParameter _arg2) in C:\Users\ridec\Code\OldManTeacher\build.fsx:line 130
        at Fake.Core.TargetModule.runSimpleInternal(TargetContext context, Target target) in D:\a\1\s\src\app\Fake.Core.Target\Target.fs:line 201
-> Win32Exception: The system cannot find the file specified
   StackTrace:
        at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
        at System.Diagnostics.Process.Start()
        at Fake.Core.Process.processStarter@409.Invoke(RawCreateProcess c, Process p) in D:\a\1\s\src\app\Fake.Core.Process\Process.fs:line 419
```

If I do `gcloud app deploy --quiet` the application deploys correctly although it takes a while (multiple minutes).

**Comment by theimowski** [comment]
https://github.com/SAFE-Stack/SAFE-template/pull/243 should fix the issue

**Comment by theimowski** [comment]
I released v1.0.1 - should be available soon, can you try that version?

**Comment by theimowski** [comment]
Closing, please reopen if issue persists
